### DISCUSSION TITLE
İbax öneri

### DISCUSSION THREAD

**Discussion by tiredcointr35** [IBAX-io/go-ibax#1815] [Soru Cevap - Türkçe]
### Clear and concise description of the problem

MOBİLE MİNİNG 

### Suggested solution

BİR MİNİNG SİSTEMİ OLMALI TELEFON VE PC YE UYUMLU CPU GÜCÜNE YADA EKRAN KARTI GÜCÜNE GÖRE MİNİNG YAPILMASINA İZİN VERİLMELİ BLUT MİNİNG SEÇENEĞİ OLABİLİR ÜCRETLİ BİR ŞEKİLDE FAKAT ALIŞILMIŞIN DIŞINDA ÖZELLİKLERLE DİGER KULLANICILARIN ONUNE GECİLMELİ ÖRNEĞİN TWİTTER RT ETKİNLİKLERİNE O UYGULAMA ÜZERİNDEN DEVAM EDİLMELİ SENKRONİZE MİNİNG DESTEKLENMELİ BİRDEN ÇOK TELEFONLA BİR KULLANICININ MİNİNG YAPMASINA İZİN VERİLMELİ AYRICA BİR KAÇ OYUN OLABİLİR OYUNLARDA ÖDÜL OLARAK DAHA HIZLI KAZIM YAPILMASINI SAĞLAYAN GÜÇLENDİRİCİLER OLABİLİR FİKRİMİ DESTEKLERSENİZ SEVİNİRİM 
BÖYLE BİR ECOSİSTEM DE FİKİR BABASİ OLMAK BİLE BENİM İÇİN ŞEREFTİR 

### Alternative

YENİLİKÇİ BİR MİNİNG UYGULAMASI GERÇEĞİ YANSITAN VE KULLANICIYA VERDİĞİ DEĞERİ ÖN PLANDA TUTAN

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by tiredcointr35** [comment]
Senkronize mining olmalı kesinlile

**Comment by NurullahCelikTR** [comment]
https://www.youtube.com/watch?v=FTVKQaHzhvM

**Comment by oznuraygann** [comment]
Cüzdan ile ilgili ciddi bir sorun var belirten listede ismi olan birçok kişinin key adresleri ile giriş yapamama sorunu var cüzdan adreslerini kurtarma ile ilgili bir çözüm uretilmeli benim de dahil olduğum bu sorunda elimde key olmasına rağmen her girdiğimde yeni bir cüzdan oluşturması sorunu özellikle çözülmeli aynı keyfi girdigimde ya kabul etmiyor yada kabul ettiğinde yeni bir cüzdan oluşturuyor ve oluşturulan cuzdanla ibax cüzdanı birbirine bağlantılı olmuyor bir çok kişi mağdur olucaktir bu durumda listede benimde ismim var ama aynı sorunla karşılaştığım için cüzdan içine erişim sağlayamiyorum daha önce cüzdana erişim saglamistim bunun ekran görüntüleri var elimde save deyip telefona indirmiştim o da bulunuyor ama yinede boş buna muhakkak çözüm uretilmeli
